test/use-swr-immutable.test.tsx

```diff
@@ -52,7 +52,7 @@ describe('useSWR - immutable', () => {
     // mount <Component/> by clicking the button
     fireEvent.click(screen.getByText('mount component'))
 
-    // wait for rerender
+    // wait for rerenderinging
```

Comment:
we use rerender as a noun here which implies the behavior, same as below
```suggestion
    // wait for rerender
```